packages/notion-client.ts

```diff
@@ -0,0 +1,29 @@
+import { Client } from '@notionhq/client';
```

Comment:
Also should be moved to `packages/notion/client.ts`. Só pra centralizar mais as coisas relacionadas ao notion.

Reply:
done